pkg/docker/docker.go

```diff
@@ -43,9 +43,7 @@ type docker struct {
 	broker     *broker.Broker[*dockerContainerStatus]
 }
 
-var DockerMgr = newDockerManager()
-
-func newDockerManager() DockerManager {
+func NewDockerManager() DockerManager {
 	ctx, ctxCancel := context.WithCancel(context.Background())
```

Comment:
```suggestion
	ctx, ctxCancel := context.WithCancel(context.TODO())
```

Use context.TODO() when you haven't wired up proper context handling yet.

Reply:
Ooh good catch.

I think I need to revisit the context handling in this manager... it was written before I started using them properly 🙈